From 5809f81cd76caa3430cc647f2b7307d078da02ca Mon Sep 17 00:00:00 2001
From: Zhangbin Tong <zebulun.tong@rock-chips.com>
Date: Wed, 8 Aug 2018 18:45:16 +0800
Subject: [PATCH] ARM: rockchip_defconfig: Enable CONFIG_DM_VERITY_AVB

This is needed to enable the AVB specific dm-verity error handler.

Change-Id: Ic367b266d0e39232a6303eb59c82b9b69fc1b9cc
Signed-off-by: Zhangbin Tong <zebulun.tong@rock-chips.com>
---
 arch/arm/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/rockchip_defconfig b/arch/arm/configs/rockchip_defconfig
index bfd39a11a2c3..fd1ef2e1466b 100644
--- a/arch/arm/configs/rockchip_defconfig
+++ b/arch/arm/configs/rockchip_defconfig
@@ -244,6 +244,7 @@ CONFIG_DM_CRYPT=y
 CONFIG_DM_UEVENT=y
 CONFIG_DM_VERITY=y
 CONFIG_DM_VERITY_FEC=y
+CONFIG_DM_VERITY_AVB=y
 CONFIG_NETDEVICES=y
 CONFIG_TUN=y
 # CONFIG_NET_VENDOR_ARC is not set
-- 
2.35.3

